extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -190,6 +201,54 @@ export class AmpAdExit extends AMP.BaseElement {
       this.user().error(TAG, 'Invalid JSON config', e);
       throw e;
     }
+
+    const ampAdResourceId = getAmpAdResourceId(this.element, this.win.top);
+    if (!ampAdResourceId) {
+      this.user().error(TAG, 'No friendly parent amp-ad element was found' +
```

Comment:
this.user seems wrong?  Can we just combine this:

const ampAdResourceId = user().assert(ampAdResourceId, `${TAG}: No friendly... `);

Reply:
Changed to assert, here and in amp-analytics.